### PR TITLE
fix(docs): ASI Alliance wallet terms fixes

### PR DESCRIPTION
**Pull request by FelixNicolaeBucsa**
## Proposed Changes

_[Fetch Wallet -> ASI Alliance Wallet]_

## Linked Issues

_[1075](https://github.com/fetchai/docs/issues/1075)_

## Types of changes

_What type of change does this pull request make (put an `x` in the boxes that apply)?_

- [x] Content update.
- [ ] Bug fix (non-breaking change that fixes an issue).
- [ ] New feature added (non-breaking change that adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to stop working as expected).
- [ ] Something else (e.g., tests, scripts, example, deployment, infrastructure).

## Checklist

_Put an `x` in the boxes that apply:_

- [x] I have read the [CONTRIBUTING](https://github.com/fetchai/docs/blob/main/CONTRIBUTING.md) guide
- [x] Checks and tests pass locally

## Further comments

_[if this is a relatively large or complex change, kick off a discussion by explaining why you chose the solution you did, what alternatives you considered, etc...]_

### CODE REVIEW

**Review by gautamgambhir97 — APPROVED** [review]

File: pages/guides.mdx [hunk 1]
```diff
@@ -300,7 +300,7 @@ import { AgentsIcon, AgentverseIcon, CliIcon,FetchNetwork, AsiIcon, LedgerIcon }
         path: "/guides/fetch-network/fetch-wallet/mobile-wallet/gov-proposal"
       },
       {
-        title: "Fetch Wallet Hardware Connection Guide",
+        title: "ASI Alliance Wallet Hardware Connection Guide",
         path: "/guides/fetch-network/fetch-wallet/fetch-wallet-hardware-connection-guide"
```
Comment (devjsc) [inline pages/guides.mdx]: Need to update paths to, and update redirects to reflect these changes

**Review by devjsc — CHANGES_REQUESTED** [review]

File: pages/guides.mdx (hunk 1 above)
Comment (FelixNicolaeBucsa) [reply pages/guides.mdx]: not clear what you are requiring. Link is working. You mean: /guides/fetch-network/fetch-wallet/fetch-wallet-hardware-connection-guide -> /guides/fetch-network/fetch-wallet/asi-wallet-hardware-connection-guide ?

Comment (gautamgambhir97) [reply pages/guides.mdx]: @FelixNicolaeBucsa, I believe we need to update the path, as the title is now ASI Alliance Wallet. The path should be:

`/guides/fetch-network/asi-alliance-wallet/asi-alliance-wallet-hardware-connection-guide`

Comment (FelixNicolaeBucsa) [reply pages/guides.mdx]: Mmmh i see. Will change it and see where else we have this path across docs

Comment (devjsc) [reply pages/guides.mdx]: For every link that has been updated can you please update next config to setup redirects? 

like..
```
      {
        source: "/guides/agentverse/registering-agent-services",
        destination:
          "/guides/agentverse/agentverse-functions/registering-agent-services",
        permanent: true,
      },
```

Comment (FelixNicolaeBucsa) [reply pages/guides.mdx]: Have added all redirects in there for all edited links for wallet, both web and mobile

**Pull request merged**